site/docs/components/TransactionWrapper.tsx

```diff
@@ -46,10 +46,14 @@ export default function TransactionWrapper({
     console.error('TransactionWrapper:', error);
   }
 
+  function onSuccess(response: any) {
+    console.log('TransactionWrapperSuccessHandler', response)
```

Comment:
@abcrane123 Let's revisit in the next PR, what's our `console.log` dev experience strategy.